lib/odata_duty/edms.rb

```diff
@@ -54,6 +86,14 @@ def self.property_type
       'Edm.DateTimeOffset'
     end
 
+    OAS_TYPE = { 'type' => 'string', 'format' => 'date-time' }.freeze
```

Comment:
Did you mean to define it twice? Wouldn't there be only one defination after the file is loaded?

Reply:
can see from here, but each constant is inside a different class.